### PR TITLE
Feature/post list plugin

### PR DESCRIPTION
**Pull request by msawicki**
## What does this do/fix?

Integrates the post list plugin. @ryanurban feel free to update anything here that needs to be. This should be integrated with the Post List and Card Grid.

### CODE REVIEW

**Review by ryanurban — APPROVED** [review]
AWESOME! Thanks so much @msawicki ❤️ 🌮

File: wp-content/plugins/core/src/Blocks/Types/Post_List/Post_List.php [hunk 1]
```diff
@@ -10,35 +10,15 @@
 use Tribe\Libs\ACF\Repeater;
```
Comment (dpellenwood) [inline wp-content/plugins/core/src/Blocks/Types/Post_List/Post_List.php:10]: I would really like to propose that we delete this Block entirely. It's not a "real" block as defined by the designs. It was build as a demo for the Post List field (Query) functionality. The purpose of this block is generally confusing for developers on projects.

**Review by dpellenwood — APPROVED** [review]
This is so wonderful, Mike. Thank you fro tackling it. Some day I will get a chance to poke at the JS. But for now, it's a huge improvement over our existing functionality.

**Comment by dpellenwood** [comment]
@sc0ttkclark Any chance you would have the time to update the Content Loop Block to use The new PostList field after this merges? You can see the changes in the Card Grid here.

File: wp-content/plugins/core/src/Blocks/Types/Post_List/Post_List.php (hunk 1 above)
Comment (ryanurban) [reply wp-content/plugins/core/src/Blocks/Types/Post_List/Post_List.php:10]: Good call, agreed

**Comment by ryanurban** [comment]
I just pushed the post list block removal. This should be ready to merge.

File: composer.json [hunk 2]
```diff
@@ -81,6 +86,7 @@
     "mailgun/mailgun-php": "^2.8",
     "moderntribe/acf-image-select": "dev-master",
     "moderntribe/tribe-libs": "3.4.0",
+    "moderntribe/tribe-acf-post-list-field": "dev-main",
```
Comment (defunctl) [inline composer.json:89]: @msawicki want to update this to "^1.0"?

**Review by defunctl — CHANGES_REQUESTED** [review]

**Review by defunctl — APPROVED** [review]

**Comment by ryanurban** [comment]
I'm gonna do one last thing here and update the remaining blocks now merged to use this new field before we merge 😎

**Pull request merged**